packages/vscode-extension/src/handlers.ts

```diff
@@ -856,6 +866,31 @@ export async function listCollaborator(env: string): Promise<TreeItem[]> {
   return result;
 }
 
+export async function checkPermission(env: string): Promise<boolean> {
+  let result = false;
+  try {
+    const checkCoreRes = checkCoreNotEmpty();
+    if (checkCoreRes.isErr()) {
+      throw checkCoreRes.error;
```

Comment:
If it throw error, what will happen on the tree view

Reply:
result will be assigned false, and thus it will return false. The icon should not be displayed